gallery/src/components/NxButton/NxButtonIconExample.tsx

```diff
@@ -12,7 +12,7 @@ import { NxButton, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 const NxButtonIconExample = () =>
   <div className="nx-btn-bar">
     <NxButton>
-      <NxFontAwesomeIcon icon={faSync}/>
+      <NxFontAwesomeIcon aria-hidden={false} aria-label='icon label' icon={faSync}/>
```

Comment:
I don't think this icon should have a label.  It is redundant with the adjacent text.  Only in icon-only buttons was the lack of label an issue.

Reply:
5e921c6360b0259ddb2c519f27d920dbbcf1214d